Added Precise Elevation Calculation (#1655)
OttoWinter/esphomedocs: components/sun.rst

- **on_sunrise** (*Optional*, :ref:`Automation <automation>`): An automation to perform at sunrise
  when the sun crosses a specified angle.

>   - **elevation** (*Optional*, float): The elevation to cross. Defaults to 0° (horizon) (more precisely -0.833° due to atmospheric refraction).

Reviewer: ```suggestion
  - **elevation** (*Optional*, float): The elevation to cross. Defaults to -0.833° (the horizon, slightly less than 0° to compensate for atmospheric refraction).
```

- **on_sunset** (*Optional*, :ref:`Automation <automation>`): An automation to perform at sunset
  when the sun crosses a specified angle.

>   - **elevation** (*Optional*, float): The elevation to cross. Defaults to 0° (horizon) (more precisely -0.833° due to atmospheric refraction).

Reviewer: ```suggestion
  - **elevation** (*Optional*, float): The elevation to cross. Defaults to -0.833° (the horizon, slightly less than 0° to compensate for atmospheric refraction).```

  ``sunset``.
- **name** (**Required**, string): The name of the text sensor.
- **elevation** (*Optional*, float): The elevation to calculate the next sunrise/sunset event

>   for. Defaults to ``0°`` (more precisely ``-0.833°`` due to atmospheric refraction).

Reviewer: ```suggestion
  for. Defaults to -0.833° (the horizon, slightly less than 0° to compensate for atmospheric refraction).
```
